kong/reports.lua

```diff
@@ -143,6 +157,15 @@ local function send_report(signal_type, t, host, port)
   if not ok then
     return
   end
+  if tls then
+    local opts = {
+      ssl_verify = false,
+    }
+    local res, err = sock:tlshandshake(opts)
+    if not res then
+      return error("Failed to complete TLS handshake. ", err)
```

Comment:
I am not sure should this error. We have had issues before when anonymous logging has been turned on, and then that feature just started to crash whole Kong. That's why we should not fail here. People might be running Kong in internal networks with no internet connectivity with anonymous reports on.

Reply:
sounds reasonable. removing